compiler/src/dmd/init.d

```diff
@@ -11,22 +11,14 @@
 
 module dmd.init;
 
-import core.stdc.stdio;
```

Comment:
leave this one for the commented out `printf`s

Reply:
That was the point I tried to make in the description: there are no commented printfs, it's just AST nodes with constructors and some helper functions. These are almost never inspected.

Reply:
Still, Walter explicitly asked us not to remove `core.stdc.stdio` imports.

Reply:
....

Reply:
Re-added.